core/src/main/java/com/ericsson/bss/cassandra/ecchronos/core/CASLockFactory.java

```diff
@@ -199,17 +206,40 @@ private CASLockFactory(final Builder builder)
 
         myUuid = hostId;
 
-        myLockCache = new LockCache(this::doTryLock);
+        int lockTimeInSeconds = getDefaultTimeToLiveFromLockTable();
```

Comment:
This should be documented in the UPGRADE.md. Also please check other docs if stuff like lock time etc. is mentioned there.